packages/user/Host/plugin/common/src/lib.rs

```diff
@@ -49,15 +71,15 @@ fn do_get(app: String, endpoint: String) -> Result<HttpResponse, Error> {
 impl Admin for HostCommon {
     fn get_active_app() -> String {
         check_caller(
-            &["accounts", "staged-tx"],
+            &["host", "accounts", "staged-tx"],
```

Comment:
It's not a security problem, but I don't think you needed to add `host` here. Everywhere you ask for the active app, you're just using `Supervisor::get_active_app` directly.